### ISSUE TITLE
fix docker-compose.yml

### ISSUE THREAD

**Issue by immaroot** [AdrienKamran/soen390-team12#2]
Indentations are not proper and will cause an error trying to build.

**Comment by immaroot** [comment]
The error happens on branch main, while doing ``` docker-compose build```
```
ERROR: yaml.parser.ParserError: while parsing a block mapping
  in "./docker-compose.yml", line 1, column 1
expected <block end>, but found '<block mapping start>'
  in "./docker-compose.yml", line 15, column 3
```

**Comment by immaroot** [comment]
fixed in commit eca2349e3c21689eeaab3447f3f2ed80a5ece18d
